config/buses/gcppubsub/README.md

```diff
@@ -3,10 +3,20 @@
 Deployment steps:
 1. Setup [Knative Eventing](../../../DEVELOPMENT.md)
```

Comment:
It probably makes more sense to point to installing the core from a release, rather than development head: https://github.com/knative/docs/tree/master/eventing#installation